myapp/spec/system/tasks_spec.rb

```diff
@@ -135,9 +135,9 @@
 
   describe 'show tasks list ordered by specific column in ascending/descending order' do
     before do
-      create(:task, name: 'Task1', priority: 3, status: 2)
-      create(:task, name: 'Task2', priority: 1, status: 3)
-      create(:task, name: 'Task3', priority: 2, status: 1)
+      create(:task, name: 'Task1', priority: 3, status: 2, expired_date: '2023-06-30')
```

Comment:
[FYI]
looks like 'status: 2' is 'magic number'.
can I do this ?

```ruby
ex: status: 2 → status: :done
```

Reply:
I changed them to show the string instead of magic number here: https://github.com/Fablic/training/pull/1566/commits/4bd972abffc53416b89887584a28236458f4b434
Please kindly check it~